### DISCUSSION TITLE
Problema para inscribirse a la primera mesa de Algo 2

### DISCUSSION THREAD

**Discussion by MartinFarres** [harpomaxx/algoritmos2#39] [General]
Hola profe, queria saber como hacer por si queriamos inscribirnos para rendir Algo 2 en esta primer mesa ya que, no vamos a tener la regularidad hasta el viernes y las inscripciones cierran hoy.
Desde ya muchas gracias

**Comment by MartinFarres** [comment]
@harpomaxx

**Comment by harpomaxx** [comment]
Hola Martin, lamentablemente nosotros no vamos a tener las notas antes del
viernes. Por lo que no encuentro como podrían inscribirse en la primera
mesa. Al menos desde nuestro lado no hay mucho que podamos hacer. Son los
tiempos que tenemos, lamentablemente nuestras clases son los martes y
viernes.

**Reply by MartinFarres** [reply]
Y no hay manera de q hablemos con mesas, de manera presencial, y nos inscribamos, con la condicion de q regularicemos el viernes. De no regularizar, q se nos desinscriba.

**Reply by harpomaxx** [reply]
Desconozco. Se puede plantear obviamente...
